src/diffusers/utils/hub_utils.py

```diff
@@ -68,6 +69,13 @@
 MODEL_CARD_TEMPLATE_PATH = Path(__file__).parent / "model_card_template.md"
 SESSION_ID = uuid4().hex
 
+_CLASS_REMAPPING_DICT = {
```

Comment:
I think this can live in `modeling_utils.py`?

Reply:
Used in the `configuration_utils.py`, too.